src/client/client_impl.cc

```diff
@@ -115,18 +123,23 @@ google::protobuf::util::StatusOr<ReadFileChunkReply> ClientImpl::ReadFileChunk(
 
   // If file chunk metadata is not available, issue an OpenFileRequest to get
   // all information needed
-  if (!file_chunk_metadata_in_cache) {
+  if (!file_chunk_metadata_in_cache || force_get) {
```

Comment:
you use "force_get" because the cache is outdated (no longer a lease holder), so if we are trying to not use the cache at all, should we skip calling cache manager above to avoid unnecessary work?

Reply:
Yes, correct, I was trying to keep the change minimal. If force_get is set then we should skip the above, I will add that part.